src/main/sensors/voltage.h

```diff
@@ -67,7 +67,9 @@ typedef enum {
 #define VBAT_MULTIPLIER_MIN 1
 #define VBAT_MULTIPLIER_MAX 255
 
+#ifndef VBAT_LPF_FREQ
```

Comment:
This is bound to break targets once we move to universal targets. If this should be adjustable for targets, make it a runtime option.

Reply:
sure, this PR is to fix compilation issues.  I can do a new PR at a later date for that work.

Reply:
it is broken, it isn't a regression. it's been since the SPRacingF4NEO target file was modified NOT to include the correct value for VBAT_LPF_FREQ.  Nit sure when that happened but i noticed the issue when building CF which does have the correct VBAT_LPF_FREQ setting fir the F4EVO.

Reply:
commit removed from PR, no-longer needed.